### PR TITLE
github: Enable dependabot

### PR DESCRIPTION
**Pull request by sameo**
For automatic tracking of dependencies and version updates.

### CODE REVIEW

**Review by Xynnn007 — APPROVED** [review]
LGTM, thanks!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: cargo
```
Comment (Xynnn007) [inline .github/dependabot.yml:3]: Do we need an indent here? As https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#example-of-a-basic-setup-for-three-package-managers shows, the whole `package-ecosystem` object should be one of the list of values ​​of `updates`.

Comment (sameo) [reply .github/dependabot.yml]: I'm not sure if it is mandatory, but it is more readable for sure. Fixed now.

**Comment by sameo** [comment]
With github-actions updates now as well.

**Pull request merged**